l10n_ec_account_edi/data/email_template_unauthorized_notify.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_unauthorized_notify" model="mail.template">
+        <field name="name">Unauthorized electronic documents</field>
+        <field name="model_id" ref="model_account_edi_document" />
```

Comment:
I see an inconsistency between the model used in the email template(`model_account_edi_document`), and the record passed in the function executed by cron job when rendering the template. The error I get is this:
![image](https://github.com/OCA/l10n-ecuador/assets/7775116/e544b5b3-1664-4408-a167-94ce2307a836)

Model must be `Companies(res.company)`
![image](https://github.com/OCA/l10n-ecuador/assets/7775116/83b35914-f957-45fd-b74c-06494f44a657)